### ISSUE TITLE
Compatibility with Redmine 3.2

### ISSUE THREAD

**Issue by topkool** [martin-denizet/redmine_login_audit#22]
I got an issue, but the log shows that:

~~~
Processing by LoginAuditController#index as HTML
  Current user: abatista (id=8)
  Rendered plugins/redmine_login_audit/app/views/login_audit/index.html.erb within layouts/admin (5.7ms)
Completed 500 Internal Server Error in 174ms (ActiveRecord: 160.4ms)

ActionView::Template::Error (Asset was not declared to be precompiled in production.
Add `Rails.application.config.assets.precompile += %w( wice_grid.js )` to `config/initializers/assets.rb` and restart your server):
    1: <% content_for :header_tags do %>
    2:     <%= stylesheet_link_tag 'login_audit', :plugin => 'redmine_login_audit' %>
    3:     <%= stylesheet_link_tag 'wice_grid', :plugin => 'redmine_login_audit' %>
    4:     <%= javascript_include_tag 'wice_grid' %>
    5:     <script type="text/javascript">
    6:         function filterByDate(value) {
    7:             $('#grid_f_date option[value=' + value + ']').prop('selected', true);
  app/helpers/application_helper.rb:1224:in `javascript_include_tag'
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
~~~

Therefore, I added `Rails.application.config.assets.precompile += %w( wice_grid.js )` to `config/initializers/assets.rb` (I had to create the file) and restarted my server, and it works.

**Comment by nnmalex** [comment]
I can confirm I had the same issue on 3.2 and I fixed it the same was as well.

**Comment by martin-denizet** [comment]
Thanks, will check it out

**Comment by msneet** [comment]
I have the same issue, but method above didn't work for me. 
My environment is:
Environment:
  Redmine version                3.2.0.stable
  Ruby version                   2.1.8-p440 (2015-12-16) [x86_64-linux]
  Rails version                  4.2.5
  Environment                    production
  Database adapter               Mysql2

**Comment by nmilanov** [comment]
Same error here:

```
Processing by LoginAuditController#index as HTML
  Current user: nmilanov (id=19)
  Rendered plugins/redmine_login_audit/app/views/login_audit/_login_audit_grid.html.erb (2.1ms)
  Rendered plugins/redmine_login_audit/app/views/login_audit/index.html.erb within layouts/admin (6.7ms)
Completed 500 Internal Server Error in 20ms (ActiveRecord: 2.0ms)

ActionView::Template::Error (WiceGrid: Column 'api' is not found in table 'login_audits'! If 'api' belongs to another table you should declare it in :include or :join when initialising the grid, and specify :model in column declaration.):
    16:     format_time(audit.created_on)
    17:   end
    18:
    19:   g.column name: t(:label_la_api), in_csv: false, attribute: 'api' do |audit|
    20:     checked_image(audit.api?)
    21:   end
    22:
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'

```

Here some env:
```
Environment:
  Redmine version                3.2.0.stable
  Ruby version                   2.2.4-p230 (2015-12-16) [x86_64-linux]
  Rails version                  4.2.5
  Environment                    production
  Database adapter               PostgreSQL
```

I'm looking forward for a fix

**Comment by nmilanov** [comment]
I had the same issue on 3.2 but I can't fix it like @topkool  and @nnmalex .
After I added `Rails.application.config.assets.precompile += %w( wice_grid.js ) to config/initializers/assets.rb`

I get another error:
```
Processing by LoginAuditController#index as HTML
  Current user: nmilanov (id=19)
  Rendered plugins/redmine_login_audit/app/views/login_audit/_login_audit_grid.html.erb (3.6ms)
  Rendered plugins/redmine_login_audit/app/views/login_audit/index.html.erb within layouts/admin (38.4ms)
Completed 500 Internal Server Error in 51ms (ActiveRecord: 3.1ms)

ActionView::Template::Error (WiceGrid: Column 'api' is not found in table 'login_audits'! If 'api' belongs to another table you should declare it in :include or :join when initialising the grid, and specify :model in column declaration.):
    16:     format_time(audit.created_on)
    17:   end
    18:
    19:   g.column name: t(:label_la_api), in_csv: false, attribute: 'api' do |audit|
    20:     checked_image(audit.api?)
    21:   end
    22:
  lib/redmine/sudo_mode.rb:63:in `sudo_mode'
```
And then I changed the table `login_audits` - see 3 files in `./plugins/redmine_login_audit/db/migrate`
Then restart apache2 server.
It's not the best way to make it run - but it's running now

My environment is:
```
Environment:
  Redmine version                3.2.0.stable
  Ruby version                   2.2.4-p230 (2015-12-16) [x86_64-linux]
  Rails version                  4.2.5
  Environment                    production
  Database adapter               PostgreSQL
```

**Comment by martin-denizet** [comment]
@nmilanov I think you forgot to migrate the database.

**Comment by nmilanov** [comment]
I did 
`rake db:migrate RAILS_ENV=production`

**Comment by nmilanov** [comment]
@martin-denizet you are right
I execute:
`rake db:migrate_plugins RAILS_ENV=production`
work perfectly

**Comment by martin-denizet** [comment]
Fixed in version 0.2.3
